### DISCUSSION TITLE
[STAC] Enable the Viewer Page to retrieve filter parameters from the URL

### DISCUSSION THREAD

**Discussion by drnextgis** [developmentseed/eoAPI#138] [General]
It would be helpful if there was an option to incorporate the filter value in the URL when configuring a filter, allowing for easy sharing of the result of the Viewer Page (index.html)

**Comment by vincentsarago** [comment]
@drnextgis which viewer are your referring to?

**Reply by drnextgis** [reply]
https://github.com/developmentseed/eoAPI/blob/main/runtime/eoapi/stac/eoapi/stac/templates/stac-viewer.html
